tests/core/models/test_work.py

```diff
@@ -1,8 +1,11 @@
 import datetime
+from contextlib import nullcontext
 from unittest.mock import MagicMock
 
+import opensearchpy
 import pytest
 import pytz
+from _pytest.logging import LogCaptureFixture
```

Comment:
Minor: I noticed just today that `pytest` started exporting these symbols for use. So we can import for type checking `from pytest import LogCaptureFixture` to avoid the private package. I was thinking I'd make a PR to clean this up in the code base, might be worth changing here.

Reply:
Made a quick PR for this since I was thinking about it:
https://github.com/ThePalaceProject/circulation/pull/1777